frontend/src/components/SwitchLanguage.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const SwitchLanguage = () => {
+  const { i18n } = useTranslation();
+
+  return (
+    <div className="col-2">
+      <select
+        className="form-select mb-3"
+        onClick={(e) => i18n.changeLanguage(e.target.value)}
+      >
+        <option value="hu" selected>
```

Comment:
Mindig a magyar a selected? A select szerintem lehetne controlled component az i18n által

`<select value={value} onChange={handleChange}>`

i18n.language-el lehet elkérni az aktuálisan beállított nyelvet.